src/BaGet.Core/Configuration/BaGetOptions.cs

```diff
@@ -38,6 +38,11 @@ public class BaGetOptions
         /// </summary>
         public bool IsReadOnlyMode { get; set; } = false;
 
+        /// <summary>
+        /// The port the baGet server will run on.
```

Comment:
Should we link to the documentation to explain the supported URL formats here? Specifically, this documentation: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host?view=aspnetcore-3.1#server-urls